CHANGELOG.md

```diff
@@ -27,3 +27,4 @@ and this project adheres to
 - ci: cross-compile binaries to improve build time
 - test: update integration tests to use prepopulated Katana dumped state
 - ci: add `CHANGELOG.md` and enforce it is edited for each PR on `main`
+- feat: implemented Net Api methods
```

Comment:
Changelog should be bottom to up, i.e the most recent tweet should go up.

Reply:
Done!